<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterForm</class>
 <widget class="QWidget" name="FilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widgetFilter" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Площадь: </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1">
        <item>
         <widget class="QCheckBox" name="checkBox_after">
          <property name="text">
           <string>от:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_after">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_before">
          <property name="text">
           <string>до:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_before">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Статус:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_status">
          <item>
           <property name="text">
            <string>Любой</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Свободно</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>В аренде</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Не доступно</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Арендатор:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_rentor"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>56</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset">
          <property name="text">
           <string>Сброс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_apply">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ok">
          <property name="text">
           <string>Ок</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_after</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_after</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_before</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox_before</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
